config/eventing-istio.yaml

```diff
@@ -4,30 +4,29 @@ config:
   branches:
     "release-v1.8":
       openShiftVersions:
-        - 4.13
-        - 4.10
+        - version: 4.13
```

Comment:
Yes. Aside from "version", one can also define cron schedule and onDemand for this version of openshift:
E.g.
```
config:
  branches:
    main:
      openShiftVersions:
        - version: 4.13
          cron: "0 4 * * *"
        - version: 4.11
          cron: "0 6 * * *"
          onDemand: true
```
Any suggestions how to make it less verbose? I left the element one level above as "openShiftVersions" but it could also be "openShifts".

Reply:
Ah, I haven't seen the `cron` one. I think its fine with knowing this.